### DISCUSSION TITLE
How to set zoom level in settings.json just like VScode?

### DISCUSSION THREAD

**Discussion by Zhou-Yicheng** [zed-industries/zed#6565] [Feature Requests]
`"window.zoomLevel"`

**Comment by aexvir** [comment]
zed currently doesn't support ui scaling, but the buffer size can have its font size configured via the `buffer_font_size` option
to change that, go to zed > settings > open settings (or cmd+,) and enter
```
"buffer_font_size": 14,
```
or whatever is the size that you'd like

**Reply by iamnbutler** [reply]
Yeah, this is correct. We will build UI scaling at some point but we need to go back and refactor everything that deals with UI scale. I'd like to do this soon but no immediate timeline.

**Comment by JosephTLyons** [comment]
You can now customize UI font and size with:

- `ui_font_size`
- `ui_font_family`

**Reply by topwebmaster** [reply]
That didnt work todays
